data-raw/instream/stanislaus/stainislaus_river.rmd

```diff
@@ -15,39 +15,42 @@ library(scales)
 ```
 
 ## Future Data Improvements
-[Future data improvement bullets]
+None Planned 
 
-## Instream Spawning and Rearing Habiat
+## Instream Spawning and Rearing Habitat
 
-**Data Source:** [Spawning Aceituno 1993 (FWS) PHABSIM model and the rearing from the FWS River2D model.] TODO
-[Need to add links to souce documents TODO
-<https://flowwest.github.io/cvpiaHabitat/reference/stanislaus_river_instream.html>]
+**Data Source:** [Aceituno 1993](https://cvpiahabitat-r-package.s3-us-west-2.amazonaws.com/cvpia-sit-model-inputs/Aceituno1993_StanislausRiverIFIMStudy.pdf) (pg 19)
+[FWS 2012](https://cvpiahabitat-r-package.s3-us-west-2.amazonaws.com/cvpia-sit-model-inputs/CVPIA_Annual_Progress_Report_Fiscal_Year_2012.pdf) (pg 6)
 
-Description of modeling TODO
 
-Instream spawning and rearing habitat for Fall Run Chinook Salmon and Steelhead in the Stanislaus River is based on data from [data source description]. The [DWR FERC instream spawning and rearing habitat data] was provided by [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service in a spreadsheet (see '~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx'). Mark Gard instructed us to use I4:N60 of the 'Stanislaus' tab within his 'IFIMWUA.xlsx' These cells represent the sums of the rearing habitat in the canyon segment, knights ferry to orange blossom, orange blossom to jacomb myers, and jacob myers to san joaquin and then converted from sqft to sqft/1000ft. Spawning is modeled
-as one 50 mi segment.
+Instream spawning and rearing habitat for Fall Run Chinook Salmon and Steelhead in the Stanislaus River is based on data from Aceituno 1993 and FWS 2012. The Aceituno 1993 study uses the PHABSIM model which was developed in 1989 to look at the relationship between physical habitat availability and streamflow for Chinook Salmon Spawning in approximately 24 miles of the Stanislaus.THe original spawning data is on pg 19 of the report. THe rearing data comes from the 2012 FWS CVPIA Annual progress report.The FWS used four sites on the river and conducted 2-D hydraulic modeling of all the sites a graph representing the rearing totals for all the sites is on pg 6 of the report. The data was provided to us by [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service in a spreadsheet (see '~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx'). Mark Gard instructed us to use I4:N60 of the 'Stanislaus' tab within his 'IFIMWUA.xlsx' These cells represent the sums of the rearing habitat in the canyon segment, knights ferry to orange blossom, orange blossom to jacomb myers, and jacob myers to san joaquin and then converted from sqft to sqft/1000ft. Spawning is modeled
+as one 50 mi segment (REPORT SAYS @$ MILES***THIS MIGHT BE THE REASON FOR THE DATA CONFUSION.
```

Comment:
did you figure out the units issue?

Reply:
Still need to check with Mark/Mark Gard to figure out issue. Commented in project issue to keep track of.